resources/lib/vrtplayer/vrtapihelper.py

```diff
@@ -32,10 +32,12 @@ def __init__(self, kodi_wrapper):
 
     def get_tvshow_items(self, path=None):
         if path:
-            api_url = self._VRTNU_SUGGEST_URL + '?facets%5Bcategories%5D=' + path
+            params = {'facets[categories]': path}
+            api_url = '?'.join((self._VRTNU_SUGGEST_URL, urlencode(params)))
```

Comment:
It just executes faster. https://stackabuse.com/python-performance-optimization/#stringconcatenation

Reply:
I doubt it makes a real difference for non-lists (i.e. a few strings).
From a readability perspective, in the above case it may appear we are concatenating multiple strings using a question mark, which is something that triggers me as I would not expect it. (Obviously, here it's only using 2 arguments)

But we are using python, so speed is already secondary to readability and rapid development anyway. If we really want to make a difference here, we should really be looking at a statically typed alternative. 

(On that subject I can wholeheartedly recommend this material at https://github.com/patvarilly/dsghent-python-for-data-scientists-2016, especially https://github.com/patvarilly/dsghent-python-for-data-scientists-2016/blob/master/notebooks/10_CythonAndNumba.ipynb, a real eye-opener for me)

Reply:
You're right, in this case execution time when using `+` is the same.

Reply:
The real performance hit is actually looping and then concatenating, which is what all those examples were doing.